handbook/people-ops/benefits-and-perks.md

```diff
@@ -2,13 +2,17 @@
 
 We want our teammates to be happy, healthy, and productive.
 
+## 🏡 All remote
+
+We are an [all-remote team](../../company/remote/index.md) so you never waste time commuting again, have flexible work hours, and work with the best people worldwide. We give you a [budget to either work from a coworking space or setup your home office](spending-company-money.md).
```

Comment:
```suggestion
We are an [all-remote team](../../company/remote/index.md) so you have flexible work hours, work with the best people worldwide, and never waste time commuting. We give you a [budget to either work from a coworking space or setup your home office](spending-company-money.md).
```

otherwise "never" might seem to apply to the other things